projects/packages/jitm/src/class-jitm.php

```diff
@@ -55,9 +55,9 @@ public function register() {
 		 * @since 3.7.0
 		 * @since 5.4.0 Correct docblock to reflect default arg value
 		 *
-		 * @param bool false Whether to show just in time messages.
+		 * @param bool true Whether to show just in time messages.
 		 */
-		if ( ! apply_filters( 'jetpack_just_in_time_msgs', false ) ) {
+		if ( ! apply_filters( 'jetpack_just_in_time_msgs', true ) ) {
```

Comment:
Do we then need to set it to `true` in Jetpack?
https://github.com/Automattic/jetpack/blob/3d1d9678cb73e9f1d6b71b8cc83bfc4946c5b964/projects/plugins/jetpack/class.jetpack.php#L726-L727

Reply:
I removed that and the `add_filter( 'jetpack_just_in_time_msg_cache', '__return_true', 9 );` call in commit 01ce5cc90e.